diff --git a/dist/index.js b/dist/index.js
index c71c2558d027d30b084ee949524a1c4b34995c3c..930f18b08565a501f287afb4273ece3354e2192a 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -107698,6 +107698,7 @@ class MoviPlayer extends j {
     if (!this.stateManager.is("idle") && !A2) throw new Error("Player must be idle to load");
     A2 && (this.config.source = A2, this.stateManager.getState()), this.stateManager.setState("loading"), this.emit("loadStart", void 0), this.lastBufferedTime = 0, this.coverArt?.close?.(), this.coverArt = null, this.destroyPreviewPipeline();
     const t2 = this.config.source, n2 = !this.config.sourceAdapter && t2 && "url" === t2.type && t2.url ? t2.url : null, b2 = n2?.toLowerCase() ?? "";
+    this.config.subtitleTracks && this.config.subtitleTracks.length > 0 && (this._subtitleTracks = [...this.config.subtitleTracks]);
     if (n2 && (b2.includes(".m3u8") || b2.includes(".mpd") || b2.includes(".ism"))) {
       const A3 = b2.includes(".m3u8"), j2 = b2.includes(".mpd"), s2 = A3 ? "HLS" : b2.includes(".ism") ? "Smooth Streaming" : "DASH";
       this.trackManager.on("videoTrackChange", (A4) => {
